db/models/group.js

```diff
@@ -4,7 +4,10 @@ const { STRING } = require('sequelize');
 
 module.exports = db =>
   db.define('groups', {
-    name: STRING,
+    name: {
+      type: STRING,
+      allowNull: false
```

Comment:
We need a
```
validate: {
    notEmpty: true
}
```
as well as allowNull anytime we don't want users to submit an empty string.